cypress/integration/upload.spec.ts

```diff
@@ -41,9 +41,6 @@ describe('Upload', () => {
           link: imageUrl
         }
       })
-      cy.fixture('acme.png').then(image => {
-        this.image = image
-      })
```

Comment:
Any reason you removed this to copy it into the two other tests? Did it make problems with the execution of the tests?

Reply:
It made problems and according to the cypress docs you shouldn't do that.